mantidimaging/main.py

```diff
@@ -7,16 +7,29 @@
 from mantidimaging import helper as h
 from mantidimaging.core.configs import recon_config
 from mantidimaging.core.utility import ExecutionTimer
+from mantidimaging.core.utility.optional_imports import safe_import
 
 formatwarning_orig = warnings.formatwarning
 warnings.formatwarning = \
     lambda message, category, filename, lineno, line=None: \
     formatwarning_orig(message, category, filename, lineno, line='')
 
 
+def startup_checks():
+    tomopy = safe_import('tomopy')
+    if tomopy is not None:
+        ver = tomopy.__version__
+        if ver != '1.1.2':
```

Comment:
Do we want to specify an exact version of a minimum? (`distutils.LooserVersion` is useful here).

Reply:
We specify an exact version in the `environment.yml` and Conda package `meta.yml`, this is just mirroring the version that is pinned there.

Reply:
Fair point 👍